config/settings.yml

```diff
@@ -141,11 +141,11 @@
     :image_name: manageiq/embedded-ansible
     :image_tag: latest
     :service_account: miq-privileged
-:ems:
```

Comment:
This nil value for a key that's overridden by sub-settings causes those settings to be lost.

Reply:
As an example, this setting is overriding the ems key for other configuration sources: such as the [amazon provider](https://github.com/ManageIQ/manageiq-providers-amazon/blob/71267026170ae25338b25a114fa56f18592b3ece/config/settings.yml#L2)